pkg/k8s/third_party.go

```diff
@@ -82,7 +82,13 @@ func (r *CiliumNetworkPolicy) Parse() (api.Rules, error) {
 					if retRule.Ingress[i].FromEndpoints[j].HasKeyPrefix(labels.LabelSourceReservedKeyPrefix) {
 						continue
 					}
-					retRule.Ingress[i].FromEndpoints[j].MatchLabels[labels.LabelSourceK8sKeyPrefix+PodNamespaceLabel] = namespace
+					// Store the same namespace on the "FromEndpoints" map
```

Comment:
Let's reword this comment:
The user can explicitly specify the namespace in the FromEndpoints selector. If omitted, we limit the scope to the namespace the policy lives in.